cosmic-core/scripts/src/main/resources/scripts/vm/hypervisor/kvm/send_config_properties_to_systemvm.py

```diff
@@ -41,6 +42,7 @@ def write_guest_file(path, content):
     except Exception as ex:
         print Exception, ":", ex
         print "ERROR: Something went wrong, exiting."
+        time.sleep(15)
```

Comment:
Maybe externalize the value passed to the `time.sleep(...)` function for better maintenance: `WAIT_BEFORE_EXITING=15 # seconds`?